README.md

```diff
@@ -131,6 +132,7 @@ These rules enforce some of the [best practices recommended for using Cypress](h
 | [no-async-tests](docs/rules/no-async-tests.md)                           | disallow using `async`/`await` in Cypress test cases       | ✅  |
 | [no-force](docs/rules/no-force.md)                                       | disallow using `force: true` with action commands          |    |
 | [no-pause](docs/rules/no-pause.md)                                       | disallow using `cy.pause()` calls                          |    |
+| [no-debug](docs/rules/no-debug.md)                                       | disallow using `cy.debug()` calls                          |    |
```

Comment:
```suggestion
| [no-debug](docs/rules/no-debug.md)                                       | disallow using `cy.debug()` calls                          |    |
| [no-force](docs/rules/no-force.md)                                       | disallow using `force: true` with action commands          |    |
| [no-pause](docs/rules/no-pause.md)                                       | disallow using `cy.pause()` calls                          |    |
```

The entries are supposed to be in alphabetical order.

If you follow the instructions in https://github.com/cypress-io/eslint-plugin-cypress/blob/master/CONTRIBUTING.md#document-generation then the plugin writes this section automatically.

In this case it's easiest just to correct the sort order manually since everything else is correct.

Reply:
Thanks, done.

I saw the contributing guide, but through Github Codesandbox it seemed easier to just copy paste the rule that was 99% the same. We could add a note about rules order in the guide though, it seems to be missing.